privacyidea/static/components/config/controllers/configControllers.js

```diff
@@ -73,6 +73,17 @@ myApp.controller("policyListController", ["$scope", "$stateParams", "$location",
 
     // listen to the reload broadcast
     $scope.$on("piReload", $scope.getPolicies);
+
+    $scope.confirmDeletePolicy = function (policyName) {
```

Comment:
I think it should be possible to only have one function for policies, resolvers and anything!

~~~~javascript
$scope.confirmDelete = function(delete_function, identifier, question) {
        if ($scope.deletion_confirmation) {
            if (confirm(question)) {
                delete_function(identifier);
            }
        } else {
                delete_function(identifier);
        }
    };
~~~~

Then this could be called

~~~~javascript
confirmDelete($scope.delPolicy, policyName, "Do you really want to delete this beautiful and well important policy?");
~~~~

Then we could drop all other functions `confirmDeleteResolver`, `confirmDeletePolicy`...

Reply:
fixed in 9c208ab4212d0e7fd5a0133faf6722115b3a40c9.